config/environments/development.rb

```diff
@@ -15,7 +15,18 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
-
+  # Default URL options for mailer
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+  address:              'smtp.gmail.com',
+  port:                 587,
+  #domain:               'example.com',
+  user_name:            'mikro.tutoriale',
+  password:             'HWilliams',
```

Comment:
Wywal konfiguracje do secrets.yml Tutaj jest opis o co w ogóle chodzi z secrets.yml: http://richonrails.com/articles/the-rails-4-1-secrets-yml-file